### PR TITLE
Update connect-zendesk-to-microsoft-cloud-app-security.md with change of admin permissions.

### PR DESCRIPTION
**Pull request by idbasre**

### CODE REVIEW

**Comment by PRMerger13** [comment]
@idbasre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ This article provides instructions for connecting Microsoft Cloud App Security t
 ### Configure Cloud App Security
 
 >[!NOTE]
->The Zendesk admin needs to be logged into their Zendesk organization within their browser before installing the connector from the Cloud App Security portal.
+>The Zendesk user that is configuring the integration needs to be alwayes Zendesk admin.
```
Comment (dcurwin) [inline CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md:49]: What do you mean "always needs to be a Zendesk admin"? Why can't we just say "needs to be a Zendesk admin"?

**Review by dcurwin — CHANGES_REQUESTED** [review]
Question in the comments.

File: CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md (hunk 1 above)
Comment (idbasre) [reply CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md:49]: We have seen customers that downgrade permissions of the Zendesk user after the connection. Therefore, we want to emphasize that the Zendesk user should remain an admin.

Comment (dcurwin) [reply CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md:49]: So how about, "The Zendesk user that is configuring the integration must always remain a Zendesk admin, even after the connector is installed."?

Comment (idbasre) [reply CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md:49]: @dcurwin  Sounds much better :)

**Comment by PRMerger19** [comment]
@idbasre : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by idbasre** [comment]
Commit again with the required changes.

File: CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ This article provides instructions for connecting Microsoft Cloud App Security t
 ### Configure Cloud App Security
 
 >[!NOTE]
->The Zendesk admin needs to be logged into their Zendesk organization within their browser before installing the connector from the Cloud App Security portal.
+>The Zendesk user that is configuring the integration must always remain a Zendesk admin, even after the connector is installed
```
Comment (dcurwin) [inline CloudAppSecurityDocs/connect-zendesk-to-microsoft-cloud-app-security.md:49]:
```suggestion
>The Zendesk user that is configuring the integration must always remain a Zendesk admin, even after the connector is installed.
```

**Comment by PRMerger14** [comment]
@dcurwin : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**